### ISSUE TITLE
DocX user setup and user improvements

### ISSUE THREAD

**Issue by LaurenceLeo** [hezmondo/Jinn#132]
In light of my previous issue on PDF editing it is apparent to me that none of the users know how to use DocX. This is partially because the wording in Jinn is not very userfriendly. It may be worth rewording buttons in Jinn so that users can decide correctly how to save their files.

Almost everyone is saving to PDF and has no idea what DocX means.

**Comment by jnbarchan** [comment]
The problem (with reference to your previous issue about editing PDF files instead) is that presumably the implication/purpose of this issue is still that users wish to edit the letters they generate in Jinn but now via DocX, right?

I have to say it's not ideal.  I'd rather people edited their letters whilst still inside Jinn.

Be aware that if users do edit DocX files via LibreOffice/Word, there is no "going back" to have the result intelligible/updatable within Jinn.  When Jinn saves as "DocX", it has no way of *actually* producing a native DocX file.  Rather, it just saves HTML, but names the file `.docx`.  That seems to work for both LibreOffice & Word, in that they read a `.docx` file which actually contains HTML happily.  If the user then saves, I imagine the file content gets changed into genuine DocX format.

One wonders just why users have to wait till after they have exported a letter to PDF/DocX to decide they want to start making edits to it... ?

**Comment by LaurenceLeo** [comment]
I wasn't clear in my original statement. I do **not** mean for users to edit their work in Jinn. Currently almost none of the users are aware that letters can be saved as PDF or in a format which they can revisit and edit in Libreoffice. The "save as DocX" label for buttons is confusing our users, we need to make it clear that this is an editable format they can revisit later.

In regards to your last point, the situation is where a user writes a letter to a customer then later on realises it requires editing. At this point they cannot edit the letter in PDF and have to rewrite the whole letter. This is often a problem with things like Deeds of Release, where the customers details are wrong.

In summary, the main point is that we might want to adjust our interface with saving letters in DocX so that our uneducated users know what it means.

**Comment by LaurenceLeo** [comment]
test
